_sass/tools/_mixins.scss

```diff
@@ -52,3 +52,10 @@
   justify-content: space-between;
   align-items: center;
 }
+
+@mixin display-flex-copyright {
```

Comment:
These seems to only be used once. If this is the only place it is used, then it doesn’t need to be a mixin. These styles can live in the single instance the mixin is used instead.

Reply:
I made it more general so it can be reused in other places.